src/dmd/traits.d

```diff
@@ -504,7 +504,7 @@ extern (C++) Expression semanticTraits(TraitsExp e, Scope* sc)
     {
         if (global.params.vcomplex)
         {
-            if (isTypeX(t => t.iscomplex() || t.isimaginary()))
+            if (isTypeX(t => t.iscomplex() || t.isimaginary()).value == 1)
```

Comment:
`isTypeX` returns a class instance (now `IntegerExp`) which will always be `true` ...

Reply:
Gah!

Reply:
And you couldn't have used `isTypeX(...).isBool(true)`?

Reply:
-> https://github.com/dlang/dmd/pull/7722